routes/accountsRouter.js

```diff
@@ -0,0 +1,89 @@
+const express = require("express");
+const Accounts = require("../data/helpers/accountsModel");
+
+const router = express.Router();
+
+router.get("/", async (req, res, next) => {
+  try {
+    const {limit, sortby, sortdir} = req.query;
+    const accounts = await Accounts.get(limit, sortby, sortdir);
+    res.status(200).json(accounts);
+  } catch (err) {
+    next(err);
```

Comment:
Could you explain you are calling `next()` inside an endpoint if it's not a middleware? Would love to hear your reasoning behind this. :)

Reply:
Sure, `next()` in this case passes the caught error along to my error-handling middleware. Perfectly fine to call next here!

Reply:
Thanks! That makes sense!